libs/platform/src/lib/components/list/base-list-item.ts

```diff
@@ -172,6 +227,21 @@ export class BaseListItem extends BaseComponent implements OnInit, AfterViewChec
     /**Getter and setter the list of items */
     @Input()
     set item(item: any) {
```

Comment:
plz add type

Reply:
Dont use `item:any` I assume you already know the data type you are working